mlflow/docker-compose.yaml

```
@@ -0,0 +1,12 @@
+services:
+  mlflow:
+    build: .
+    image: ghcr.io/mlflow/mlflow
+    ports:
+      - 5000:5000
```

You should add variable `MLFLOW_PORT` to the `.env` file to do that.
```suggestion
      - ${MLFLOW_PORT}:5000
```